src/components/Lend/LoanSearch/LoanSearchLocationFilter.vue

```diff
@@ -104,6 +116,11 @@ export default {
 		},
 	},
 	watch: {
+		activeIsoCodes(next, prev) {
+			console.log('activeIsoCodes next: ', next);
+			console.log('activeIsoCodes prev: ', prev);
+			console.log(mapIsoCodesToCountryNames(next, this.regions));
```

Comment:
This could be where we "apply" the actively selected items but may not be in the end...